pom.xml

```diff
@@ -35,7 +35,7 @@
   </scm>
 
   <prerequisites>
-    <maven>3.1.0-SNAPSHOT</maven>
+    <maven>${maven.version}</maven>
```

Comment:
This will need to be reverted. Upgrading the per-requisite version will prevent some people from using the plugin.

Reply:
Although this has brought to my attention that I accidentally changed this from 3.0.0 when getting this new version of the plugin for development. So I'll have to change this back to 3.0.0

Reply:
done and pushed